src/api/text.js

```diff
@@ -1,637 +1,232 @@
+const defaultFont = require('../fonts/single-line/hershey/simplex.js')
+const { rectangular_extrude } = require('./ops-extrusions')
+const { union } = require('./ops-booleans')
 
-/** Construct a with, segments tupple from a character
- * @param {Float} x - x offset
- * @param {Float} y - y offset
- * @param {Float} char - character
- * @returns {Object} { width: X, segments: [...] }
- *
- * @example
- * let charData = vector_char(0, 12.2, 'b')
- */
-function vector_char (x, y, char) {
-  char = char.charCodeAt(0)
-  char -= 32
-  if (char < 0 || char >= 95) return { width: 0, segments: [] }
+const defaultsVectorParams = {
```

Comment:
not sure I understand why the defaults are outside the function ?

Reply:
Because is shared from the two functions (avoid multiple sources of truths and save some bytes)